modules/core/src/lib/mode-handlers/mode-handler.js

```diff
@@ -70,6 +71,42 @@ export class ModeHandler {
     return null;
   }
 
+  getSelectedFeatures(): Array<Feature> {
+    const { features } = this.featureCollection.getObject();
+    return this._selectedFeatureIndexes.map(index => {
+      const feature = features[index];
+      const featureCopy: any = { ...feature };
+      // Preserve feature index in geojson properties
+      featureCopy.properties = { ...featureCopy.properties, _internalIndex: index };
```

Comment:
Do you still need `getSelectedFeatures()` and `_internalIndex`?

Reply:
No these are no longer needed. I removed `getSelectedFeatures()` and `getSelectedGeometries()` methods from `ModeHandler`.